2016/10/06 21:14:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in /home/xshadowx/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#1 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii
---
2016/10/06 21:16:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in /home/xshadowx/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#1 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii
---
2016/10/08 19:40:51 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [14] unable to open database file' in /home/xshadowx/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/xshadowx/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/xshadowx/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/xshadowx/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/xshadowx/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/xshadowx/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/xshadowx/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/xshadowx/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/xshadowx/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/xshadowx/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/xshadowx/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/xshadowx/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model/index
HTTP_REFERER=http://localhost:8082/index.php?r=gii/default/index
---
2016/10/08 19:42:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [14] unable to open database file' in /home/xshadowx/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/xshadowx/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/xshadowx/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/xshadowx/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/xshadowx/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/xshadowx/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/xshadowx/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/xshadowx/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/xshadowx/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/xshadowx/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/xshadowx/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/xshadowx/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model/index
HTTP_REFERER=http://localhost:8082/index.php?r=gii/default/index
---
2016/10/08 19:42:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [14] unable to open database file' in /home/xshadowx/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/xshadowx/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/xshadowx/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/xshadowx/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/xshadowx/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/xshadowx/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/xshadowx/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/xshadowx/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/xshadowx/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/xshadowx/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/xshadowx/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/xshadowx/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model/index
HTTP_REFERER=http://localhost:8082/index.php?r=gii/default/index
---
2016/10/08 19:42:03 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [14] unable to open database file' in /home/xshadowx/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/xshadowx/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/xshadowx/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/xshadowx/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/xshadowx/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/xshadowx/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/xshadowx/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/xshadowx/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/xshadowx/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/xshadowx/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/xshadowx/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/xshadowx/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model/index
HTTP_REFERER=http://localhost:8082/index.php?r=gii/default/index
---
2016/10/08 19:42:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [14] unable to open database file' in /home/xshadowx/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/xshadowx/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/xshadowx/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/xshadowx/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/xshadowx/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/xshadowx/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/xshadowx/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/xshadowx/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/xshadowx/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/xshadowx/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/xshadowx/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/xshadowx/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model/index
HTTP_REFERER=http://localhost:8082/index.php?r=gii/default/index
---
2016/10/08 19:43:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [14] unable to open database file' in /home/xshadowx/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/xshadowx/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/xshadowx/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/xshadowx/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/xshadowx/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/xshadowx/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/xshadowx/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/xshadowx/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/xshadowx/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/xshadowx/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/xshadowx/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/xshadowx/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model/index
HTTP_REFERER=http://localhost:8082/index.php?r=gii/default/index
---
2016/10/08 19:43:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [14] unable to open database file' in /home/xshadowx/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/xshadowx/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/xshadowx/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/xshadowx/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/xshadowx/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/xshadowx/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/xshadowx/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/xshadowx/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/xshadowx/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/xshadowx/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/xshadowx/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/xshadowx/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model/index
HTTP_REFERER=http://localhost:8082/index.php?r=gii/default/index
---
2016/10/08 19:51:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "comment".' in /home/xshadowx/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('comment')
#1 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=comment
---
2016/10/09 16:04:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in /home/xshadowx/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#1 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php
---
2016/10/09 16:05:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in /home/xshadowx/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#1 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2016/10/09 16:09:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in /home/xshadowx/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#1 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/
---
2016/10/09 16:12:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in /home/xshadowx/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('')
#1 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php
---
2016/10/09 16:17:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/default/index".' in /home/xshadowx/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/default/ind...')
#1 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#2 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii/default/index
HTTP_REFERER=http://localhost:8082/index.php?r=gii/default/login
---
2016/10/09 16:17:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [14] unable to open database file' in /home/xshadowx/www/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /home/xshadowx/www/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /home/xshadowx/www/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /home/xshadowx/www/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /home/xshadowx/www/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/xshadowx/www/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /home/xshadowx/www/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/xshadowx/www/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/xshadowx/www/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/xshadowx/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/xshadowx/www/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/xshadowx/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/xshadowx/www/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /home/xshadowx/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /home/xshadowx/www/yii/framework/base/CApplication.php(185): CWebApplication->processRequest()
#15 /home/xshadowx/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model/index
HTTP_REFERER=http://localhost:8082/index.php?r=gii/default/index
---
